R/arc-select.R

```diff
@@ -237,12 +284,35 @@ collect_layer <- function(x, n_max = Inf, token = Sys.getenv("ARCGIS_TOKEN"), ..
 
 # utility -----------------------------------------------------------------
 
+#' Check if x is a FeatureLayer or Table object
+#'
```

Comment:
Please add useful description here for why this function exists and what its intended use is for other developers (me 😉 )